MinimedKit/PumpEvents/DailyTotal523PumpEvent.swift

```diff
@@ -33,7 +33,7 @@ public struct Sara6EPumpEvent: PumpEvent {
     
     public var dictionaryRepresentation: [String: AnyObject] {
         return [
-            "_type": "Sara6E",
+            "_type": "DailyTotal523",
```

Comment:
You're right, I thought I remember a midnight event but all I see summary-wise is what shows up in the CSV at the time of the history dump (today units delivered, yesterday units delivered). 

> On Sep 8, 2016, at 9:03 PM, Pete Schwamb <notifications@github.com> wrote:
> 
> In MinimedKit/PumpEvents/DailyTotal523PumpEvent.swift:
> 
> > @@ -33,7 +33,7 @@ public struct Sara6EPumpEvent: PumpEvent {
> >      
> >      public var dictionaryRepresentation: [String: AnyObject] {
> >          return [
> > -            "_type": "Sara6E",
> > +            "_type": "DailyTotal523",
> Yeah, I did some searching through the CSVs and I don't see anything that looks like the daily summary either.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>